roles/confluent.variables/defaults/main.yml

```diff
@@ -558,10 +584,10 @@ ksql_monitoring_interceptors_enabled: "{{ monitoring_interceptors_enabled }}"
 
 # Control Center Variables
 
-### Only use to customize Linux User Control Center Service runs with. User must exist on host.
+### Only use to customize Linux User Control Center Service runs with. Default user is cp-control-center
 control_center_user: "{{control_center_default_user}}"
 
-### Only use to customize Linux Group Control Center Service user belongs to. Group must exist on host.
+### Only use to customize Linux Group Control Center Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Control Center Service user belongs to. Default group is confluent.